### ISSUE TITLE
Make Size function O(1) time complexity

### ISSUE THREAD

**Issue by Joshcarp** [anz-bank/decimal#24]
Size gets the magnitude of the significand,
currently using a for loop method that divides till it reaches zero. 
@anzdaddy speculates that it might be possible to get the same result using bitwise inspection as used in the renormalise function, although this method currently doens't work for all decimal numbers
```
numBits := 64 - bits.LeadingZeros64(significand)
	numDigits := numBits * 3 / 10
	normExp := 15 - numDigits 

```

**Comment by anzdaddy** [comment]
Something like: `if significand >= powerOf10[numDigits+1] { numDigits++ }`

**Comment by Joshcarp** [comment]
solved in commit 2880f17
